src/services/authService.js

```diff
@@ -6,6 +6,10 @@ const logout = () => {
   return Promise.resolve();
 };
 
+const saveLinkedInToken = token => {
```

Comment:
Don't need this function since it's not doing anything else other than returning TokenUtils. Use that directly